### DISCUSSION TITLE
Update folder name with no date

### DISCUSSION THREAD

**Discussion by ramo21** [Official-Husko/NN-Downloader#46] [Ideas]
Would it be possible to make the file names be configurable, so that the folder don't have to have dates in the name, if the user does not want it. I run into the issue where I want to download more/update for the same tag but a new folder gets created and ends up with duplicate files in the different folders.

**Comment by Official-Husko** [comment]
Sure, i can look into it. As an example a simple "use dates in folder name" in the config with true or false and if disabled it just throws new or updated images into the already existing folder?

**Reply by ramo21** [reply]
Exactly, if "use dates in folder name" is false the folder name would be just the tag and if true the folder would be date with tag. Thanks!
